src/config/constants/farms.ts

```diff
@@ -649,7 +649,7 @@ const farms: FarmConfig[] = [
       97: '',
       56: '0xC743Dc05F03D25E1aF8eC5F8228f4BD25513c8d0',
     },
-    token: tokens.blk,
+    token: tokens.blink,
```

Comment:
Branched from https://github.com/pancakeswap/pancake-frontend/pull/713 to fix the compile errors. This diff will disappear once that's merged